src/BagnoDB/Serialization.fs

```diff
@@ -1,7 +1,11 @@
 namespace BagnoDB
 
     open MongoDB.Bson.Serialization
+    open MongoDB.Bson.Serialization.Serializers
 
     module Serialization =
         let bson bsonSerializer =
-            BsonSerializer.RegisterSerializationProvider bsonSerializer
\ No newline at end of file
+            BsonSerializer.RegisterSerializationProvider bsonSerializer
+
+        let registerDecimalSerializer () =
```

Comment:
A unit test that checks if it works okay, would be great :)

Reply:
See overall comment for full details